react/features/filmstrip/functions.web.js

```diff
@@ -174,6 +177,33 @@ export function calculateThumbnailSizeForVerticalView(clientWidth: number = 0) {
     };
 }
 
+/**
+ * Calculates the size for thumbnails when in vertical view layout
+ * and the filmstrip is resizable.
+ *
+ * @param {number} clientWidth - The height of the app window.
+ * @param {number} filmstripWidth - The width of the filmstrip.
+ * @returns {{local: {height, width}, remote: {height, width}}}
+ */
+export function calculateThumbnailSizeForResizableVerticalView(clientWidth: number = 0, filmstripWidth: number = 0) {
+    const horizontalMargin = VERTICAL_FILMSTRIP_MIN_HORIZONTAL_MARGIN + SCROLL_SIZE
```

Comment:
I'm not sure if it make sense to merge this function with `calculateThumbnailSizeForVerticalView`?

Anyway I also see the value in having 2 different functions but could you please at least extract the `horizontalMargin` to be a top level constant? It is exactly the same for both functions, isn't it?

Reply:
I wouldn't merge them. I think it's more readable like this. I'll extract the horizontal margin tough